
Creating window glfw
Epochs: 0, Loss: 442058016.0, lr: 0.01
Traceback (most recent call last):
  File "tv_value_synthesis_trajopt_cartpole.py", line 214, in <module>
    renderer.render(traj[:, selection, 0, :sim_params.nq].cpu().detach().numpy())
  File "/home/daniel/Repos/Mujoco_Python_Sandbox/value_gradient/mj_renderer.py", line 27, in render
    self.viewer.render()
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/mujoco_py/mjviewer.py", line 206, in render
    render_inner_loop(self)
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/mujoco_py/mjviewer.py", line 182, in render_inner_loop
    super().render()
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/mujoco_py/mjviewer.py", line 60, in render
    glfw.poll_events()
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/glfw/__init__.py", line 1854, in poll_events
    _glfw.glfwPollEvents()
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/glfw/__init__.py", line 687, in errcheck
    _reraise(exc[1], exc[2])
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/glfw/__init__.py", line 70, in _reraise
    raise exception.with_traceback(traceback)
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/glfw/__init__.py", line 666, in callback_wrapper
    return func(*args, **kwargs)
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/mujoco_py/mjviewer.py", line 330, in key_callback
    img = self._read_pixels_as_in_window()
  File "/home/daniel/.local/share/virtualenvs/Mujoco_Python_Sandbox-gNKKWzd6/lib/python3.7/site-packages/mujoco_py/mjviewer.py", line 222, in _read_pixels_as_in_window
    self.sim.render(resolution[0], resolution[1])
  File "mjsim.pyx", line 156, in mujoco_py.cymj.MjSim.render
  File "mjsim.pyx", line 158, in mujoco_py.cymj.MjSim.render
  File "mjrendercontext.pyx", line 46, in mujoco_py.cymj.MjRenderContext.__init__
  File "mjrendercontext.pyx", line 114, in mujoco_py.cymj.MjRenderContext._setup_opengl_context
  File "opengl_context.pyx", line 130, in mujoco_py.cymj.OffscreenOpenGLContext.__init__
RuntimeError: Failed to initialize OpenGL